Makefile

```diff
@@ -21,11 +21,11 @@ all: fmt lint test build
 
 .PHONY: build
 build:
-	scripts/build.sh
+	ENV=dev scripts/build.sh
```

Comment:
Why do we need and `ENV` var if we call two distinct scripts ?

Reply:
This value is used by goreleaser `ldflags: "-s -w -X github.com/cloudskiff/driftctl/build.env={{ .Env.ENV }}`

For some reasons, defining that value in the script does not work as expected so I added it there

Reply:
In the script you have 
```
# By default build for dev
ENV=${ENV:-"dev"}
```

I think you can replace it with 
```
ENV="dev"
# and
ENV="release"
```

as we use two distinct script now we do not need for a defaut like it was done before